mappings/net/minecraft/entity/LightningEntity.mapping

```diff
@@ -1 +1,16 @@
 CLASS app net/minecraft/entity/LightningEntity
+	FIELD b seed J
+	FIELD c ambientTick I
+	FIELD d remainingActions I
+	FIELD e cosmetic Z
+	FIELD f channeller Luv;
+	METHOD <init> (Lbdb;DDDZ)V
+		ARG 1 world
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 cosmetic
+	METHOD a spawnFire (I)V
+		ARG 1 extraFireBlocksCount
```

Comment:
`spreadAttempts`

Reply:
`spreadAttempts` is a way better name IMO